edb/server/main.py

```diff
@@ -439,7 +439,24 @@ async def run_server(
     global server
     server = server_mod
 
-    logger.info(f"starting EdgeDB server {buildmeta.get_version_line()}")
+    ver_meta = buildmeta.get_version_metadata()
+    extras = []
+    source = ""
+    if build_date := ver_meta["build_date"]:
+        nice_date = build_date.strftime("%Y-%m-%dT%H:%MZ")
+        source += f" on {nice_date}"
+    if ver_meta["scm_revision"]:
+        source += f" from revision {ver_meta['scm_revision']}"
+        if source_date := ver_meta["source_date"]:
+            nice_date = source_date.strftime("%Y-%m-%dT%H:%MZ")
+            source += f" ({nice_date})"
+    if source:
+        extras.append(f", built{source}")
+    if ver_meta["target"]:
+        extras.append(f"for {ver_meta['target']}")
```

Comment:
What's the point of open-coding `get_version_line()` here?

Reply:
This is a partial revert of https://github.com/edgedb/edgedb/pull/5115, it was open-coding before #5115; but we can still keep the `get_version_line()` here.

Reply:
I would keep it